docs/Home.md

```diff
@@ -7,6 +7,6 @@ This wiki is the official documentation. Use the list of pages on the right to n
 
 Theos was initially “iphone-framework”, a project created to simplify building code at the command line for iOS devices (primarily jailbroken devices). It later underwent significant changes and became Theos, a flexible [Make](https://www.gnu.org/software/make/)-based build system primarily for jailbreak software development, but also with complete support for building for other supported platforms. Theos runs on, and can build projects for, macOS, iOS, Linux, and Windows (under [Cygwin](https://cygwin.com/) or [Windows Subsystem for Linux](https://msdn.microsoft.com/commandline/wsl/about)).
 
-As of 2015, the project stalled as Dustin Howett was unable to devote time to it. It was since picked up by [the community](https://github.com/theos/theos/graphs/contributors), adding many desired features and fixing issues. The legacy version of Theos that used to be located at DHowett/theos is permanently preserved on the [legacy](https://github.com/theos/theos/tree/legacy) branch.
+In 2015, the project stalled as Dustin Howett was unable to devote time to it. Since then, it has been taken up by [the community](https://github.com/theos/theos/graphs/contributors) who have added a host of desired features and fixed numerous issues. The legacy version of Theos that used to be located at DHowett/theos is permanently preserved on the [legacy](https://github.com/theos/theos/tree/legacy) branch.
 
-There is also documentation available for the original Theos at the [iPhone Dev Wiki](http://iphonedevwiki.net/index.php/Theos), and [theos-ref](https://github.com/theiostream/theos-ref); however be aware that not all information is updated for the changes Theos has undergone recently.
\ No newline at end of file
+There is also documentation available for the original Theos at the [iPhone Dev Wiki](http://iphonedevwiki.net/index.php/Theos) and [theos-ref](https://github.com/theiostream/theos-ref). Please note, however, that not all of said documentation is up-to-date and may not reflect recent changes to Theos.
```

Comment:
Heh, it’s not “recent” any more I guess. This homepage is terrible at introducing Theos and really needs a rewrite 😬

Probably not something for this PR though. If you have any ideas for updating these two paragraphs without rewriting the whole page though, feel free to give it a shot.

Reply:
I don't know...I feel like it does a pretty good job of briefly explaining the purpose, history, and evolution of Theos, which is all it really needs to do, in my opinion. 

The only thing that _may_ be worth adding in the future is a list of noteworthy features that would entice someone to use it even if their deployment target isn't jailbroken iOS.